apps/bandwagon/forms.py

```diff
@@ -143,6 +149,15 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
+        # Check the honeypot here instead of 'clean_your_name' so the
+        # error message appears at the top of the form in the __all__ section
+        if self.cleaned_data['your_name']:
+            statsd.incr('collections.honeypotted')
```

Comment:
in addition to this, I'd suggest also adding a log statement because grepping the logs could be helpful